src/ImageSharp/Metadata/Profiles/IPTC/IptcProfile.cs

```diff
@@ -242,7 +246,7 @@ public void UpdateData()
             int i = 0;
             foreach (IptcValue value in this.Values)
             {
-                this.Data[i++] = 28;
```

Comment:
I wonder whether we should add a little ascii table just indicating what the dataset tags look like?